config/serverless.oblt.yml

```diff
@@ -12,111 +12,21 @@ xpack.legacy_uptime.enabled: false
-      # Discover `All` feature privilege should implicitly grant `All` access to Observability app.
-      all.composedOf:
-        - feature: "observability"
-          privileges: [ "all" ]
-      # Discover `Read` feature privilege should implicitly grant `Read` access to Observability app.
-      read.composedOf:
-        - feature: "observability"
-          privileges: [ "read" ]
-  ### Fleet feature privileges are fine-tuned to grant access to Logs app.
-  fleetv2:
-    privileges:
-      # Fleet `All` feature privilege should implicitly grant `All` access to Logs app.
-      all.composedOf:
-        - feature: "logs"
-          privileges: [ "all" ]
-      # Fleet `Read` feature privilege should implicitly grant `Read` access to Logs app.
-      read.composedOf:
-        - feature: "logs"
-          privileges: [ "read" ]
-  ### Infrastructure feature privileges are fine-tuned to grant access to Logs, and Observability apps.
-  infrastructure:
-    ### By default, this feature named as `Metrics`, but should be renamed to `Infrastructure`.
-    name: "Infrastructure"
-    privileges:
-      # Infrastructure's `All` feature privilege should implicitly grant `All` access to Logs and Observability apps.
-      all.composedOf:
-        - feature: "logs"
-          privileges: [ "all" ]
-        - feature: "observability"
-          privileges: [ "all" ]
-      # Infrastructure's `Read` feature privilege should implicitly grant `Read` access to Logs and Observability apps.
-      read.composedOf:
-        - feature: "logs"
-          privileges: [ "read" ]
-        - feature: "observability"
-          privileges: [ "read" ]
-  ### Logs feature is hidden in Role management since it's automatically granted by either Infrastructure, or Applications features.
-  logs.hidden: true
+  discover_v2.category: 'observability'
+  ### By default, this feature named as `Metrics`, but should be renamed to `Infrastructure`.
+  infrastructure.name: 'Infrastructure'
   ### Machine Learning feature should be moved from Analytics category to the Observability one and renamed to `AI Ops`.
   ml:
-    category: "observability"
+    category: 'observability'
     order: 1200
-  ### Observability feature is hidden in Role management since it's automatically granted by either Discover,
-  ### Infrastructure, Applications, Synthetics, or SLOs features.
-  observability.hidden: true
-  ### SLOs feature privileges are fine-tuned to grant access to Observability app.
-  slo:
-    privileges:
-      # SLOs `All` feature privilege should implicitly grant `All` access to Observability app.
-      all.composedOf:
-        - feature: "observability"
-          privileges: [ "all" ]
-      # SLOs `Read` feature privilege should implicitly grant `Read` access to Observability app.
-      read.composedOf:
-        - feature: "observability"
-          privileges: [ "read" ]
-  ### Stack alerts is hidden in Role management since it's not needed.
-  stackAlerts.hidden: true
```

Comment:
Remove `stackAlerts.hidden`. We want the stack rules experience to match stateful. See additional comment in the `navigation_tree.ts` file.

Reply:
After discussion with product, we've decided to go back to hiding stackAlerts. cc: @jasonrhodes